# HG changeset patch
# Parent  7abf6025cbad6273be3dc47cd2e2efc51f260c29
Properly initialize the process title even when no bulbs are discovered

diff --git a/core/lightsd.c b/core/lightsd.c
--- a/core/lightsd.c
+++ b/core/lightsd.c
@@ -295,6 +295,10 @@
 
     lgtd_lifx_discovery_start();
 
+    // update at least once: so that if no bulbs are discovered we still get a
+    // clear status line.
+    lgtd_daemon_update_proctitle();
+
     event_base_dispatch(lgtd_ev_base);
 
     lgtd_cleanup();
diff --git a/docs/first-steps.rst b/docs/first-steps.rst
--- a/docs/first-steps.rst
+++ b/docs/first-steps.rst
@@ -92,7 +92,7 @@
 Assuming you've just built :ref:`lightsd from the sources
 <build_instructions>`, lightsd will be in the ``core`` directory [#]_.
 
-The examples are communicating with lightsd through a pipe or an unix socket,
+The examples are communicating with lightsd through a pipe or an Unix socket,
 start lightsd with them:
 
 ::
@@ -179,8 +179,8 @@
 
    . share/lightsc.sh
 
-You can use the following things to send commands to your bulbs from your
-current shell or shell script:
+You can use the following variable and functions to send commands to your bulbs
+from your current shell or shell script:
 
 .. data:: LIGHTSD_COMMAND_PIPE
 
@@ -241,12 +241,11 @@
 
    examples/lightsc.py
 
-From there a ``c`` variable has been initialized for you, this a small object
-let you directly execute commands on your bulb:
+From there, a ``c`` variable has been initialized for you: this small object
+lets you directly execute commands on your bulb:
 
 For example toggle your lights again:
 
-
 .. code-block:: python
 
    c.power_toggle("*")
